chapters/5-processing.tex

```diff
@@ -255,7 +254,11 @@ \subsection{Tidying raw data}
 (in Stata, often through \texttt{collapse})
 and then merging or joining those results to the firms data table.
 
-In a tidy workflow, your clean dataset is a set of one or more tidy data tables.
+In a tidy workflow, your clean dataset comes from 
+a group of one or more tidy data tables.
```

Comment:
I think the way it was is more accurate. As in saying that a "dataset is a set of one or more tidy tables" rather than it "_comes_" from it.